src/index.css

```
@@ -57,6 +57,10 @@ section#game {
     min-height: 515px;
     margin-bottom: 35px;
     position: relative;
+    -ms-user-select: none;
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    user-select: none;
```

https://caniuse.com/?search=user-select -- `user-select: none` has been supported by Chrome, Edge, and Firefox for a good amount of time now. It's just Safari that's a holdout. 

```suggestion
    -webkit-user-select: none;
    user-select: none;
```